lib/profiles/TR/CRY.js

```diff
@@ -1,14 +1,17 @@
 exports.name = 'CRY';
 exports.config = {
     status: 'CRY',
-    longStatus: 'Candidate Registry Snapshot',
+    longStatus: 'Candidate Registry',
```

Comment:
I don't understand how the tests passed with the previous longStatus. Are we even testing this?